assignments/closure.js

```diff
@@ -1,9 +1,19 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function sayHello() {
+  let name = "Jim";
+  function slang() {
+    console.log("Yo " + name)
+  }
+  slang();
+}
+console.log(sayHello());
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
+let count = 0;
+return () => (++count);
   // Return a function that when invoked increments and returns a counter variable.
 };
 // Example usage: const newCounter = counter();
```

Comment:
Use your counter to finish the exercise and demonstrate understanding about closure.